app/src/main/AndroidManifest.xml

```diff
@@ -27,17 +28,18 @@
             android:configChanges="orientation"
             android:screenOrientation="portrait"
             android:theme="@style/AppTheme.NoActionBar">
+        </activity>
+        <activity
+            android:name=".activities.Main"
+            android:configChanges="orientation"
+            android:screenOrientation="portrait">
```

Comment:
laten we dit niet in deze ticket doen aangezien deze geen authenticatie flow in de app fixt. Er is al een PR open die main als launch activity zet en de authenticatie flow regelt.